vision/api/face_detection/faces.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+# Copyright 2015 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Draws squares around faces in the given image."""
+
+import argparse
+import base64
+
+from PIL import Image
+from PIL import ImageDraw
+
+from googleapiclient import discovery
+import httplib2
+from oauth2client.client import GoogleCredentials
+
+
+# [START get_vision_service]
+DISCOVERY_URL='https://{api}.googleapis.com/$discovery/rest?version={apiVersion}'
```

Comment:
Unfortunately, yes - OP apis' endpoints are a subdomain of googleapis.com, whereas apiary ones aren't